### PR TITLE
Slider component rough first pass. Additional commit for dev cleanliness.

### PR DESCRIPTION
**Pull request by phrough**

### CODE REVIEW

**Comment by dkilgore-eightfold** [comment]
@phrough please refer to the Octuple [contribution guidelines](https://github.com/EightfoldAI/octuple/blob/main/src/CONTRIBUTING.md)

File: src/components/Slider/slider.module.scss [hunk 1]
```diff
@@ -0,0 +1,90 @@
+$sliderWidth: 200px;
+
+.sliderContainer {
```
Comment (dkilgore-eightfold) [inline src/components/Slider/slider.module.scss:3]: nit: consistency in class names, snake case vs camel. Snake is recommended.

File: src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { FC, useCallback, useEffect, useRef, useState } from 'react';
+
+import { SliderProps } from './Slider.types';
+
+import styles from './slider.module.scss';
+
+export const Slider: FC<SliderProps> = ({ value, min = 0, max = 100 }) => {
```
Comment (dkilgore-eightfold) [inline src/components/Slider/Slider.tsx:7]: Stub in basic UT

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: src/components/Slider/slider.module.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    z-index: 1;
+}
+
+.sliderRange {
+    background-color: #9fe5e1;
+    z-index: 2;
+}
+
+.sliderValue {
+    color: #666;
```
Comment (dkilgore-eightfold) [inline src/components/Slider/slider.module.scss:37]: Take note of other component use of color palette and css vars to enable theming.

File: src/CONTRIBUTING.md [hunk 4]
```diff
@@ -13,6 +13,10 @@
 
 from the project root folder, you can run:
 
+### `yarn install`
```
[the remark this answers is not in the document]
Comment (dkilgore-eightfold) [reply src/CONTRIBUTING.md:16]: Please read the section on commits, we use commit lint and you'll need to update the title of your pr to something like feat: slider: adds slider component.

[the remark this answers is not in the document]
Comment (phrough) [reply src/CONTRIBUTING.md:16]: Will do. Was unclear on the relationship between commits and the PR. Am separating the two commits out into independent PRs. Probably the preferred way I would guess?

**Comment by phrough** [comment]
Apologies. I screwed this up for lack of familiarity with the workflow. I'm going to close this PR and submit a new one with feedback addressed.

**Pull request closed without merging**